iphone/Classes/TiBluetoothUtils.m

```diff
@@ -26,6 +26,22 @@ @implementation TiBluetoothUtils
     return result;
 }
 
++ ( NSArray<NSUUID *> * _Nullable)NSUUIDArrayFromStringArray:(NSArray<id> *)array
```

Comment:
Should be `(NSArray<NSString *> *)array`, since the type must always be an `NSString`.